app/_posts/2023-06-23-kuma-2-3-0.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Kuma 2.3 release with MeshTCPRoute, GAMMA support and much more
+description: Kuma 2.3 release with MeshTCPRoute, GAMMA support and much more
+date: 2023-06-23
+headerImage: /assets/images/blog/test-header.jpg
+tags:
+  - Release
+---
+
+We’re excited to announce the release of Kuma 2.3. This new minor release gets us one step closer to releasing targetRef policies as defaults. 
+
+In order to take advantage of the latest and greatest in service mesh, we strongly suggest upgrading to Kuma 2.3. Upgrading is easy through `kumactl` or Helm.
+
+## Notable features:
+
+* 🚀 targetRef policies are now working with `MeshGateway`.
+* 🚀 `MeshTCPRoute` which enables to control traffic routing for TCP services.
+* 🚀 Helm improvements with more secure defaults. 
+* 🚀 Possibility to skip creating default policies when creating a mesh. 
+* 🚀 Performance improvements on Kubernetes. 
+* 🚀 Add hosts selection predicates to `MeshRetry`
+* 🚀 Initial GAMMA routes support. 
+* 🚀 Continuous improvements to the look and feel of the GUI 
```

Comment:
I'm not sure if items in this list should end with a dot, but we should be consistent.
```suggestion
* 🚀 Continuous improvements to the look and feel of the GUI.
```